php/pantheon/checks/plugins.php

```diff
@@ -77,11 +77,11 @@ public function run() {
 	protected function getPluginVulnerability( $plugin_slug )
 	{
 		// Get the vulnerability API token from the platform
-		$wpvulndb_api_token = getenv('PANTHEON_WPVULNDB_API_TOKEN');
+		$wpvulndb_api_token = getenv('PANTHEON_WPSCAN_API_TOKEN');
 
 		// Throw an exception if there is no token
 		if( false === $wpvulndb_api_token || empty( $wpvulndb_api_token ) ) {
-			throw new \Exception('No WP Vulnerability DB API Token. Please ensure the PANTHEON_WPVULNDB_API_TOKEN environment variable is set');
+			throw new \Exception('No WP Vulnerability DB API Token. Please ensure the PANTHEON_WPSCAN_API_TOKEN environment variable is set');
```

Comment:
The `WPSCAN_API_TOKEN` won't be defined, though; at least not at this time, for all sites. Shouldn't we convert this to a silent error? Better yet, report which plugins were checked for vulnerabilities, and which were not (status secure, vulnerable, or not checked), if that can be done easily enough.